$OpenBSD: patch-Source_setup_configure_py,v 1.1.1.1 2010/12/29 13:47:53 stsp Exp $

1) Remove -lresolv.

2) Fix broken lib auto-detection. The configure script will try to stat a
   shared library file, but we don't know the exact filename because it depends
   on the shared library version. So look for a .la file instead. These are
   present for all libraries the script is looking for.

--- Source/setup_configure.py.orig	Tue Dec 29 12:31:15 2009
+++ Source/setup_configure.py	Sun Dec 19 17:05:53 2010
@@ -433,7 +433,7 @@ LDLIBS=-L%(svn_lib_dir)s -Wl,--rpath -Wl,%(svn_lib_dir
 -lsvn_client-1 \
 -lsvn_diff-1 \
 -lsvn_repos-1 \
- -lcom_err -lresolv -lexpat -lneon
+ -lcom_err -lexpat -lneon
 
 #include pysvn_common.mak
 '''
@@ -455,7 +455,7 @@ LDLIBS=-L%(svn_lib_dir)s -Wl,--rpath -Wl,%(svn_lib_dir
 -lsvn_client-1 \
 -lsvn_diff-1 \
 -lsvn_repos-1 \
- -lresolv -lexpat -lneon
+ -lexpat -lneon
 
 #include pysvn_common.mak
 '''
@@ -477,7 +477,7 @@ LDLIBS=-L%(svn_lib_dir)s -Wl,--rpath -Wl,%(svn_lib_dir
 -lsvn_client-1 \
 -lsvn_diff-1 \
 -lsvn_repos-1 \
--lcom_err -lresolv -lexpat -lneon -lssl
+-lcom_err -lexpat -lneon -lssl
 PYSVN_CREATE_INIT_OPTION=%(pysvn_module_name)s
 
 #include pysvn_common.mak
@@ -500,7 +500,7 @@ LDLIBS=-L%(svn_lib_dir)s \
 -lsvn_client-1 \
 -lsvn_diff-1 \
 -lsvn_repos-1 \
--lcom_err -lresolv -lexpat -lneon -lssl
+-lcom_err -lexpat -lneon -lssl
 PYSVN_CREATE_INIT_OPTION=%(pysvn_module_name)s
 
 #include pysvn_common.mak
@@ -907,7 +907,7 @@ LDLIBS= \
         elif self.platform == 'cygwin':
             return '%s.dll.a' % libname
         else:
-            return '%s.so' % libname
+            return '%s.la' % libname
 
     def find_dir( self, name, kw, svn_root_suffix, base_dir_list, check_file ):
         dirname = self.__find_dir( name, kw, svn_root_suffix, base_dir_list, check_file )
